### ISSUE TITLE
Can not compile. Pathdraw is missing

### ISSUE THREAD

**Issue by andrey-vk** [kobbled/kl-paths#1]
I have following error when trying to compile rossum_example_ws:
`CRITICAL | Error occured while checking packages: Package pathforms is missing dependencies: pathdraw. Cannot continue`

**Comment by kobbled** [comment]
Hi @andrey-vk,

I pushed some changes to rossum_example_ws (commit kobbled/rossum_example_ws@461e6d0503ea4088def83aee1819add300e0e7d1 ). Test it out. I think the problem is in the "task-sync-test" example. I was getting a different error though so I don't know exactly if I fixed your issue. If you are still having problems make sure you have https://github.com/kobbled/Ka-Boost downloaded on your machine, and the environment variable `ROSSUM_PKG_PATH` pointing to where you downloaded it.

A note about the "task-sync-test". That one runs motion commands, so it is dependent on your robot cell configuration. You will have to update the registers in `move_to_home.ls`, and `move_to_pose.ls`, and set the home position in `move_to_home.ls`. There are also user definitions in `task_b.kl` that need to be changed as well. It can only run in teach mode, and does not work in auto mode.

The `kl-draw` module is in heavy development right now, so just be wary of that one for now. `kl-paths` should be usable other than the `pathforms` module right now.

If you have other questions about any of the tools I have on my github don't hesitate to ask. I desperately need to write documentation for ka-boost...

Thanks for flagging the issue!
